editor/js/Editor.js

```diff
@@ -644,16 +644,6 @@ Editor.prototype = {
 	fromJSON: function ( json ) {
 
 		var loader = new THREE.ObjectLoader();
-
-		// backwards
-
-		if ( json.scene === undefined ) {
-
-			this.setScene( loader.parse( json ) );
-			return;
-
-		}
```

Comment:
This block was introduced here 7431f1a3e07f0090804cbfd60a5b1fd8c2943ceb, five years ago. I think we can safely remove this now.